.travis.yml

```diff
@@ -24,6 +24,10 @@ env:
         - CONDA_CHANNELS='astropy'
         - ASTROPY_USE_SYSTEM_PYTEST=1
 
+        # List runtime dependencies for the package that are available as conda
+        # packages here.
+        - CONDA_DEPENDENCIES='memory-profiler'
```

Comment:
Maybe it should be an underscore instead of a hyphen in `memory-profiler`?

Reply:
whoops, thanks!